clients/python/agentic-sandbox-client/test_client.py

```
@@ -284,6 +284,55 @@ def test_termination_and_deletion(client: SandboxClient, sandbox: Sandbox, sandb
+        assert CLIENT_REQUEST_TIME_ANNOTATION in annotations, f"Expected annotation '{CLIENT_REQUEST_TIME_ANNOTATION}' missing"
+
+        timestamp_str = annotations[CLIENT_REQUEST_TIME_ANNOTATION]
+        print(f"Timestamp: {timestamp_str}")
+
+        # Verify it can be parsed
+        try:
+            dt = datetime.fromisoformat(timestamp_str)
+            assert dt.tzname() == 'UTC', "Timestamp should be in UTC"
+            print(f"Parsed datetime: {dt}")
+        except ValueError as e:
+            raise AssertionError(f"Failed to parse timestamp '{timestamp_str}': {e}")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)